src/staging/gatsby-node.js

```diff
@@ -7,35 +8,65 @@ exports.onPostBuild = ({ reporter }) => {
 
 exports.createPages = async ({ graphql, actions, reporter }) => {
   const { createPage } = actions
-
-  const result = await graphql(`
-    {
-      allStrapiProtectedArea {
-        nodes {
-          id
-          orcs
-          protectedAreaName
-          slug
-        }
-        totalCount
+  const parkQuery = `
+  {
+    allStrapiProtectedArea(filter: { orcs: { lt: 50 } }) {
```

Comment:
I believe the `lt: 50` filter was removed.  Is this here just from older code or is there another reason for the 50 limit?

Reply:
I believe it is old code. I will merge main in and see if this gets sorted out.